src/JMS/Serializer/Metadata/ClassMetadata.php

```diff
@@ -58,6 +58,9 @@ class ClassMetadata extends MergeableClassMetadata
     public $discriminatorMap = array();
     public $discriminatorGroups = array();
 
+    public $xmlDiscriminatorAttribute = false;
+    public $xmlDiscriminatorCData = true;
```

Comment:
capitalizing CD follows the convention used in PropertyMetadata::$xmlElementCData ...